front-end/src/visualization/Visualization.jsx

```diff
@@ -57,20 +79,86 @@ function Visualization() {
         return positions;
     }
 
+    function getColorForValence(valence) {
+        const colors = [
+            { valence: 0.0, color: '#FF0000' }, // Red for very angry or extremely sad
```

Comment:
We can probably switch this to a pitch dark. It makes the color gamut more consistent. I'd associate Orange/Yellow/Red in the same color range, and Purple/Blue/Black too.